perspective_reddit_bot/log_subreddit_comments.py

```diff
@@ -124,15 +124,19 @@ def log_subreddit(creds, subreddits, output_dir):
                        password=creds['reddit_password'])
   subreddit = reddit.subreddit(all_subs)
 
-  for i, comment in enumerate(comment_stream(subreddit.stream)):
+  while True:
     try:
-      print('.', end='')
-      sys.stdout.flush()
-      output_record = create_comment_output_record(comment)
-      if i % 25 == 0: print_comment(i, output_record)
-      append_records(output_path, [output_record])
+      for i, comment in enumerate(comment_stream(subreddit.stream)):
```

Comment:
The i's will reset with each exception. If we still want to print every 25, we can't use enumerate anymore. Instead we'll have to manually declare i in the while loop and increment it in the for loop, I think.

Reply:
The printing is really just to get a random sample of the comments being logged, don't think it's important to make this bit correct.